referral/templates/referral/select-referral-type.html

```diff
@@ -1,22 +1,25 @@
 {% extends "pttrack/base.html" %}
 
+{% load i18n %}
+<!-- trans finished -->
+
 {% load bootstrap3 %}
 
 {% block title %}
-Choose a referral type
+{% blocktrans %}Choose a referral type{% endblocktrans %}
 {% endblock %}
 
 {% block header %}
-<h1>Choose a Referral Type</h1>
-<p><i>Each patient should be provided referrals for ongoing care to several FQHCs from a pre-screened list of FQHCs. Specialty referral refers to any non-FQHC referral (e.g., the COH).</i></p>
+<h1>{% blocktrans %}Choose a Referral Type{% endblocktrans %}</h1>
+<p><i>{% blocktrans %}Each patient should be provided referrals for ongoing care to several FQHCs from a pre-screened list of FQHCs. Specialty referral refers to any non-FQHC referral (e.g., the COH){% endblocktrans %}.</i></p>
 {% endblock %}
 
 {% block content %}
 
 <div class="container text-center">
 	<div class="btn-group-vertical">
   	{% for referral_type in referral_types %}
-	    <a class="btn btn-{% cycle 'info' 'success' 'warning' 'danger' %}" href="{% url 'new-referral' rtype=referral_type.slugify pt_id=pt.pk %}" role="button">Create {{referral_type}} referral</a>
+	    <a class="btn btn-{% cycle 'info' 'success' 'warning' 'danger' %}" href="{% url 'new-referral' rtype=referral_type.slugify pt_id=pt.pk %}" role="button">{% blocktrans %}Create{% endblocktrans %} {{referral_type}} {% blocktrans %}referral{% endblocktrans %}</a>
```

Comment:
If I've understood the `blocktrans` docs correctly, you can do:

```
{% blocktrans %}Create {{referral_type}} referral{% endblocktrans %}
```

instead of what you have here.

Reply:
You are right, didn't catch that one